paper-header-panel.html

```diff
@@ -140,23 +154,48 @@
       position: static;
       width: 100%;
     }
-  </style>
 
-  <template>
-    <div id="outerContainer" class="layout vertical">
+    ::content paper-toolbar,
+    ::content .paper-header {
+      position: relative;
+      overflow: visible !important;
+    }
+
+    ::content paper-toolbar:after,
+    ::content .paper-header:after {
+      @apply(--paper-header-panel-shadow);
 
-      <content id="headerContent" select="paper-toolbar, .paper-header"></content>
+      -webkit-transition: opacity 0.5s, -webkit-transform 0.5s;
+      transition: opacity 0.5s, transform 0.5s;
 
-      <div id="mainPanel" class="layout vertical flex">
+      opacity: 0;
+      content: "";
 
-        <div id="mainContainer" class$="[[_computeMainContainerClass(mode)]]">
-          <content id="mainContent" select="*"></content>
-        </div>
+      width: 100%;
+      position: absolute;
+      left: 0px;
+      right: 0px;
+      z-index: 1;
```

Comment:
Should avoid using z-index here. For example, if I have a put ```<paper-header-panel>``` inside ```<paper-drawer-pane>``` then I will need to put a z-index on the drawer to make the drawer on top of the shadow.